### PR TITLE
added fetch projects api

### PR DESCRIPTION
**Pull request by meomancer**
issue #21
Added fetch projects api

### CODE REVIEW

**Comment by Gustry** [comment]
Looks good to me, thanks !

File: cadasta/api/project.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+
+class Project(NetworkMixin):
+    """Class to fetch available organization data."""
+
+    api_url = 'api/v1/projects/'
+
+    def __init__(self, on_finished=None):
+        """
+        Constructor.
```
Comment (ismailsunni) [inline cadasta/api/project.py]: Put the short description on the first line.

**Review by ismailsunni — APPROVED** [review]
LGTM. Just one minor comment.

File: cadasta/api/project.py (hunk 1 above)
Comment (Gustry) [reply cadasta/api/project.py]: Sorry, yes I think we skip this rule everywhere :(

Comment (meomancer) [reply cadasta/api/project.py]: @Gustry yes etienne, i'm still working to make it as my behaviour
still need learning this convention

File: cadasta/api/project.py [hunk 2]
```diff
@@ -24,8 +24,7 @@ class Project(NetworkMixin):
     api_url = 'api/v1/projects/'
 
     def __init__(self, on_finished=None):
-        """
-        Constructor.
+        """ Constructor.
```
Comment (Gustry) [inline cadasta/api/project.py]: Minor things, we usually don't add space after `"""`. Please read https://github.com/inasafe/inasafe/wiki/Coding-Standards#doc-strings-and-comments

**Pull request merged**